content/en/tracing/trace_collection/automatic_instrumentation/single-step-apm.md

```diff
@@ -232,34 +293,64 @@ To enable single step instrumentation with Helm:
 
 ### Enabling or disabling instrumentation for namespaces
 
-You can choose to selectively instrument specific namespaces or choose to not instrument them.
-
-To enable instrumentation for specific namespaces, add `enabledNamespaces` configuration to your `datadog-values.yaml` file:
-{{< highlight yaml "hl_lines=6-9" >}}
-      datadog:
-        apiKeyExistingSecret: datadog-secret
-        site: <DATADOG_SITE>
-        apm:
-          instrumentation:
-            enabled:true
-            enabledNamespaces: # Add namespaces to instrument
-               - namespace_1
-               - namespace_2
- {{< /highlight >}}
-
-To disable instrumentation for specific namespaces, add the `disabledNamespaces` configuration to your `datadog-values.yaml` file:
-{{< highlight yaml "hl_lines=7-9" >}}
-   datadog:
-     apiKeyExistingSecret: datadog-secret
-     site: <DATADOG_SITE>
+You can choose to enable or disable instrumentation for applications in specific namespaces. The file you need to configure depends on if you enabled Single Step Instrumentation with Datadog Operator or Helm:
```

Comment:
Let's add the following information:

1. enabledNamespaces and disabledNamespaces cannot be set at the same time. This will result in SSI configuration not being applied at all.
2. By default, `kube-system` namespace and the namespace where Datadog resources are running are not instrumented.

Reply:
For point 2, I think this is essentially covered by the note we have at the beginning of the installation section:

`<div class="alert alert-info">Single Step Instrumentation doesn't instrument applications in the namespace where you install the Datadog Agent. It's recommended to install the Agent in a separate namespace in your cluster where you don't run your applications.</div>`

What do you think?